alex_post

```diff
@@ -0,0 +1 @@
+Alex adds new post
```

Comment:
For adding new posts, add a file in `_post` directory using the same file name format as done in other posts there.